data/json/snippets/newspapers.json

```diff
@@ -128,6 +128,12 @@
         "text": {
           "str": "THE 'UNCANNY' SUCCESS STORY.  Everyone's talking about the meteoric rise of tech startup \"Uncanny\" over the last few months.  From zero to household name, the company leapt to the public eye with its, well, uncanny line of talking toys.  Before we even knew what to expect, they were Wall Street's new darling, with the fastest rise in manufacturing capacity of any company in the last thirty years.  We spoke to James Hawthorne, America's newest billionaire and the man behind the name on everyone's lips.  Interview on page 3."
         }
+      },
+      {
+        "id": "years_old_news_12",
+        "text": {
+          "str": "SOYPELUSA SUITS FIND NEW AMMO:  Following the infamous disaster of the DaiZoom product launch and the numerous lawsuits that sprung from its failure, research from Harvard's Cognitive Neuropsychology labs has brought new evidence in the ongoing litigation against the company.  We spoke with lead researcher Doctor Soren Eldridge who had this to say: \"The DaiZoom bar, beyond its notoriously bad taste, appears to cause in its consumers symptoms similar to those of a major depressive disorder.  There is an actual tangible impact on mood that lasts well beyond the initial consumption of even just a few bars in a short span of time - we suspect DaiZoom may actually be chemically inhitibiting dopamine receptors.\"  Doctor Eldridge stated that further research would be needed to determine the full extent of the issue and whether consumption of DaiZoom had long-term health effects similar to existing dopamine blocking drugs.  The full interview can be found on page 7."
```

Comment:
```suggestion
          "str": "SOYPELUSA SUITS FIND NEW AMMO:  Following the infamous disaster of the DaiZoom product launch and the numerous lawsuits that sprung from its failure, research from Harvard's Cognitive Neuropsychology labs has brought new evidence in the ongoing litigation against the company.  We spoke with lead researcher Doctor Soren Eldridge who had this to say: \"The DaiZoom bar, beyond its notoriously bad taste, appears to cause in its consumers symptoms similar to those of a major depressive disorder.  There is an actual tangible impact on mood that lasts well beyond the initial consumption of even just a few bars in a short span of time - we suspect DaiZoom may actually be chemically inhibiting dopamine receptors. 
This is unprecedented, and quite exciting, given that it consists entirely of otherwise biochemically bland ingredients.\"  Doctor Eldridge stated that further research would be needed to determine the full extent of the issue and whether consumption of DaiZoom had long-term health effects similar to existing dopamine blocking drugs, and if there is any role for it as a medical therapy in the future.  The full interview can be found on page 7."
```